packages/salesforcedx-vscode-core/src/commands/forceGenerateFauxClasses.ts

```diff
@@ -42,10 +78,32 @@ class ForceGenerateFauxClassesExecutor extends SfdxCommandletExecutor<{}> {
     ForceGenerateFauxClassesExecutor.isActive = true;
     const cancellationTokenSource = new vscode.CancellationTokenSource();
     const cancellationToken = cancellationTokenSource.token;
-
     const execution = new LocalCommandExecution(this.build(response.data));
 
-    this.attachExecution(execution, cancellationTokenSource, cancellationToken);
+    channelService.streamCommandOutput(execution);
+
+    if (this.showChannelOutput) {
+      channelService.showChannelOutput();
+    }
+
+    notificationService.reportCommandExecutionStatus(
+      execution,
+      cancellationToken
+    );
+
+    let progressLocation = vscode.ProgressLocation.Notification;
+    const refreshSource = response.data;
+    if (refreshSource !== SObjectRefreshSource.MANUAL) {
+      progressLocation = vscode.ProgressLocation.Window;
+      telemetryService.sendAutomaticSObjectRefreshEvent(refreshSource);
```

Comment:
Is this okay as a separate telemetry event?

Reply:
Specific to telemetry, I think we should use `this.logMetric` and modify the command text we send when this is triggered outside of the command palette (e.g. startup, etc). Because this will currently cause 2 different metrics for one execution (here and on line 121).